src/style.css

```
@@ -0,0 +1,132 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  box-sizing: border-box;
+  color: rgba(6, 11, 53, 1);
+}
+
+body {
```

it's better to set `box-sizing` for all elements
```suggestion
  color: rgba(6, 11, 53, 1);
}

* {
  box-sizing: border-box;
}

body {
```